src/downloadlistwidget.cpp

```diff
@@ -375,6 +380,7 @@ bool DownloadListWidgetDelegate::editorEvent(QEvent *event, QAbstractItemModel *
               menu.addAction(tr("Visit on Nexus"), this,SLOT(issueVisitOnNexus()));
             }
 
+            menu.addAction(tr("Open File"), this, SLOT(issueOpenFile()));

```

Comment:
add checks to make sure the download is in the READY state. We don't want to give the user ability to open a still downloading file

Reply:
Already inside `if (state >= DownloadManager::STATE_READY)` clause.

Reply:
Yup, you are right. I didn't look at the surrounding code, my bad